Examples/CodePushDemoApp/package.json

```diff
@@ -9,6 +9,7 @@
     "babel-preset-react-native-stage-0": "1.0.1",
     "react": "15.4.0",
     "react-native": "0.40.0",
-    "react-native-code-push": "file:../../"
+    "react-native-code-push": "file:../../",
+    "react-native-windows": "git://github.com/ReactWindows/react-native-windows.git"
   }
```

Comment:
Since they are already locking to a specific version of react-native, you should also lock to a specific version of react-native-windows (these need to be kept in sync).